---
title: "Customizing CRISPR–Cas PAM specificity with protein language models"
authors: Nayfach S, Bhatnagar S, Novichkov A, Kim N, Hoffnagle AM, Hussain R, **Estevam GO**, Hill E, Ruffolo JA, Silverstein RA, Gallagher J, Kleinstiver BP, Meeske AJ, Cameron P,& Madani A
journal: Nature Biotechnology
pub_date: "2026-02-02"
image: "/static/img/pub/2026_nayfach.png"
pmid: 39829748
pmcid: PMC11741284
abstract: "CRISPR–Cas enzymes must recognize a protospacer-adjacent motif (PAM) to edit a genomic site, greatly limiting the range of targetable sequences in a genome. Although engineering strategies to alter PAM specificity exist, they typically require labor-intensive, iterative experimentation. We introduce an evolution-informed deep learning model, Protein2PAM, to efficiently guide the design of Cas protein variants tailored to recognize specific PAMs. Trained on a dataset of over 45,000 CRISPR–Cas PAMs, Protein2PAM rapidly and accurately predicts PAM specificity directly from Cas proteins across type I, II and V CRISPR–Cas systems. Using in silico mutagenesis, the model identifies residues critical for PAM recognition in Cas9 without using structural information. We use Protein2PAM to computationally evolve Nme1Cas9, generating variants with broadened PAM recognition and up to a 50-fold increase in PAM cleavage rates compared to the wild type in vitro. Our machine learning approach allows Cas enzymes to target sequences that were previously inaccessible because of PAM constraints, potentially increasing target flexibility in personalized genome editing."
#biorxiv:
biorxiv_version: "https://doi.org/10.1101/2025.01.06.631536"
#pdf:
links:
- name: "Profluent"
  url: https://www.profluent.bio/
- name: "Protein2PAM"
  url: https://protein2pam.profluent.bio/login
- name: Kleinstiver Lab @ Harvard Medical School
  url: https://www.kleinstiverlab.org/
- name: Meeske Lab @ University of Washington
  url: https://meeskelab.org/
---
